### ISSUE TITLE
puppetlabs-firewall 5.0.0 is not compatible with stdlib 9.0.0

### ISSUE THREAD

**Issue by corporate-gadfly** [puppetlabs/puppetlabs-firewall#1137]
## Describe the Bug
puppetlabs-firewall 5.0.0 is not compatible with stdlib 9.0.0

## Expected Behavior
Make a future version of puppetlabs-firewall compatible with stdlib 9.0.0

## Steps to Reproduce
1. Bump stdlib to 9.0.0
2. You will receive an error:
```
Server Error: Evaluation Error: Error while evaluating a Function Call, deprecation. ensure_packages.
This function is deprecated, please use stdlib::ensure_packages instead.
at ["/etc/puppetlabs/code/environments/qa/modules/firewall/manifests/linux/redhat.pp", 69]:["/etc/puppetlabs/code/environments/qa/site/profiles/manifests/base.pp", 12]
(file: /etc/puppetlabs/code/environments/qa/modules/firewall/manifests/linux/redhat.pp, line: 69, column: 5) on node xxx.yyy.zzz
```

## Environment
 - Node is CentOS Stream 8
 - Puppet agent 8.0.0
 - Puppet server 8.0.0

**Comment by LukasAud** [comment]
This issue should have been addressed in #1135. The module is currently pending a major release.

**Comment by mphilipps** [comment]
How long is this expected to take? Puppet 8 was released in April.

**Comment by ic248** [comment]
Agree, this is one of the only mods which doesn't support it and would be good to know the timeframe for the major release

**Comment by robertc99** [comment]
I'd like to also request a release. It would be nice to starting using stdlib 9 without getting warnings.

**Comment by corporate-gadfly** [comment]
@LukasAud : I'm not a puppet expert, but regarding #1135, why rely on `stdlib` features only available in 9.x. How about something like the following _(which will presumably work with both stdlib 8.x and 9.x)_?
```
if $package_name {
    package { $package_name:
        ensure => $package_ensure,
    }
}
```

**Comment by robertc99** [comment]
that would fail with a duplicate resource error if the package is loaded elsewhere. the entire point of ensure_packages is thats its safe against redefinitiion.

**Comment by corporate-gadfly** [comment]
TY, makes sense.

**Comment by corporate-gadfly** [comment]
Closing as #1135 has been merged and was released with 6.0.0.
